### ISSUE TITLE
vagrant-spk upgradevm fails on Windows

### ISSUE THREAD

**Issue by ocdtrekkie** [sandstorm-io/vagrant-spk#346]
It looks like it deleted my global-setup.sh but neither added the new one, nor touched the Vagrantfile.

```
PS C:\Users\jacob\source\repos\hledger> vagrant-spk upgradevm
Upgrading VM parameters in C:\Users\jacob\source\repos\hledger\.sandstorm
Traceback (most recent call last):
  File "vagrant-spk", line 1010, in <module>
  File "vagrant-spk", line 1007, in main
  File "vagrant-spk", line 667, in upgrade_vm
TypeError: a bytes-like object is required, not 'str'
[12488] Failed to execute script vagrant-spk
```

**Comment by ocdtrekkie** [comment]
I suspect the issue is https://github.com/sandstorm-io/vagrant-spk/blob/master/vagrant-spk#L671 which I do not believe works on Windows, and we probably need to *just not do* on a Windows build.

**Comment by ocdtrekkie** [comment]
Nope, it's the f.write itself, and the issue is we open it in "wb" mode instead of "w" but then write a string. Easy patch I think.

**Comment by ocdtrekkie** [comment]
Ian fixed it in #308 so I really just need a new Windows release built...

**Comment by ocdtrekkie** [comment]
https://github.com/sandstorm-io/vagrant-spk/releases/tag/v1.1.2
